docs/additional-configuration.adoc

```diff
@@ -302,7 +302,9 @@ spec:
           securityContext:
             runAsUser: 1234
 ----
-Note that the pod-overrides field does not allow configuring `containers`, `initContainers`, and `volumes`. In order to configure these elements, use existing DevWorkspace fields (container components, volume components)
+Note that the pod-overrides field does not allow configuring `containers` and `initContainers`. In order to configure these elements, use the existing DevWorkspace `spec.template.components` field.
+
+Note that extra precaution should be taken when overriding the `volumes` field. The DevWorkspace operator defines pod volumes by default for config files, metadata, and credentials. To avoid unexpected behaviour, these default volumes should not be overridden.
```

Comment:
"Operator" in "DevWorkspace Operator" is capitalized elsewhere in this doc. 

Also slight re-wording suggestion:

"The DevWorkspace Operator sets the pod volumes field by default for config files, metadata, and credentials. To avoid unexpected behaviour, the volumes field should not be overridden."

Reply:
Updated to:

> The DevWorkspace Operator sets the pod `spec.volumes` field by default for config files, metadata, and credentials. To avoid unexpected behaviour, the `spec.volumes` field should not be overridden.

WDYT?